ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.16.yaml

```diff
@@ -237,6 +237,16 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-internal-registry
+- as: azure-ipi-fullyprivate-oidc-amd-f14
```

Comment:
If possible, suggest to use arm arch to save cloud cost.

Reply:
Sure, I will update it ^^